waku/waku_api/rest/doc/schemas/apitypes.yaml

```diff
@@ -0,0 +1,189 @@
+WakuPeer:
+  type: object
+  required:
+    - multiaddr
+    - protocols
+  properties:
+    multiaddr:
+      type: string
+    protocols:
+      type: array
+      items:
+        type: object
+        required:
```

Comment:
Maybe it would be good to also show the shardInfo supported by a peer.

Reply:
Good idea!!!